website_docs/_index.md

```diff
@@ -12,19 +12,14 @@ cascade:
 weight: 24
 ---
 
-This is the OpenTelemetry for Ruby documentation. OpenTelemetry is an observability framework -- an API, SDK, and tools that are designed to aid in the generation and collection of application telemetry data such as metrics, logs, and traces.
-This documentation is designed to help you understand how to get started using OpenTelemetry for Ruby.
-
-## Status and Releases
-
-The current status of the major functional components for OpenTelemetry Ruby is
-as follows:
-
-| Traces  | Metrics | Logs    |
-| ------- | ------- | ------- |
-| Stable | Not Yet Implemented | Not Yet Implemented |
-
-{{% latest_release "ruby" /%}}
+<!--
+You can see & update the `lang_instrumentation_index_head` shortcode in
+/layouts/shortcodes/lang_instrumentation_index_head.md
+
+The data (name, status) is located at
+/data/instrumentation.yaml
+-->
```

Comment:
Let's drop the comment as we agreed to do for https://github.com/open-telemetry/opentelemetry.io/pull/1861.

Reply:
removed